diff -up libX11-1.8.12/configure.ac.autoconf269 libX11-1.8.12/configure.ac
--- libX11-1.8.12/configure.ac.autoconf269	2024-12-13 00:00:00.000000000 +0000
+++ libX11-1.8.12/configure.ac	2024-12-13 00:00:00.000000000 +0000
@@ -1,6 +1,6 @@
 
 # Initialize Autoconf
-AC_PREREQ([2.70])
+AC_PREREQ([2.69])
 AC_INIT([libX11], [1.8.12],
         [https://gitlab.freedesktop.org/xorg/lib/libx11/-/issues], [libX11])
 AC_CONFIG_SRCDIR([Makefile.am])
